runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -73,11 +79,16 @@
 	);
 
 	private static final long PRAYER_DRAIN_INTERVAL_MS = 18200;
-	private static final int CRYPT_REGION_ID = 14231;
+	static final int CRYPT_REGION_ID = 14231;
```

Comment:
This can be made `private` again as it is no longer referenced in the infobox.
```suggestion
	private static final int CRYPT_REGION_ID = 14231;
```